test/tests.py

```diff
@@ -0,0 +1,60 @@
+from operations import Operations
+from http import HTTPStatus
+import unittest
+import os
+
+# This code is probably existing as a placeholder only for testing purposes,
+# final code here will be using comparator's results. Also, won't be sending request directly to target cluster,
+# and will be using the replayer instead, so we're able to check comparator's results once the replayer outputs
+# the triples log file.
+# TODO: Add endpoints to be read from environment variable.
+
+
+class MyTestCase(unittest.TestCase):
+    def test_index(self):
+        endpoint1 = os.getenv('ENDPOINT_1', 'http://localhost:9200')
+        endpoint2 = os.getenv('ENDPOINT_2')
+        index = "my_index"
+        response1 = Operations.create_index(endpoint1, index)
+        response2 = Operations.create_index(endpoint2, index)
+        self.assert_(response1.status_code == HTTPStatus.OK and response2.status_code == HTTPStatus.OK)
```

Comment:
Please separate these so we more easily know which expectation failed when the tests fail